community/events.page.tsx

```diff
@@ -1030,6 +1030,39 @@ const events = [
     image: townHall,
     end_date: "December 11, 2024",
   },
+  {
+    name: "XRPL Meetup in London",
+    description:
+      "Calling all blockchain and XRP Ledger enthusiasts in London! Join XRPL Meetups to share knowledge, build real-life connections, and foster communities centered around blockchain and XRP Ledger. We're establishing local “XRPL Hubs” across Europe, and we want you to be a part of it!",
+    type: "meetup",
+    link: "https://xrpl.at/XRPL-Meetup-London",
+    location: "London, United Kingdom",
+    date: "January 30, 2025",
+    image:  require('../static/img/events/commons-france.png'),
```

Comment:
Going by event titles and file names, should this use the comons-london.png image?